### PR TITLE
Update release note with installer changes

### PR DESCRIPTION
**Pull request by udda1996**
## Purpose
$title

### CODE REVIEW

**Review by sm1990 — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -354,6 +354,9 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.5.0](https://git
 
 ### New features
 
+#### Installers
+- Introduced a new installer with dependencies (custom JRE) compatible with macos-arm architecture.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:358]:
```suggestion
- Introduced a new installer with dependencies (custom JRE) compatible with the macOS-ARM architecture. 
```
Shall we link to the Downloads page?

**Review by sm1990 — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -355,7 +355,7 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.5.0](https://git
 ### New features
 
 #### Installers
-- Introduced a new installer with dependencies (custom JRE) compatible with the macOS-ARM architecture. 
+- Introduced a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) with dependencies (custom JRE) compatible with the macOS-ARM architecture.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:358]:
```suggestion
- Introduced a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) by including the dependencies that are compatible with the macOS-ARM architecture. 
```

**Comment by praneesha** [comment]
@keizer619 @anupama-pathirage - Should we mention this under the `Install Ballerina` section?

**Comment by sm1990** [comment]
> @keizer619 @anupama-pathirage - Should we mention this under the `Install Ballerina` section?

@praneesha we'll removing that section in the future right?

**Comment by praneesha** [comment]
> > @keizer619 @anupama-pathirage - Should we mention this under the `Install Ballerina` section?
> 
> @praneesha we'll removing that section in the future right?

What's the reason to remove it?

**Comment by anupama-pathirage** [comment]
> > > @keizer619 @anupama-pathirage - Should we mention this under the `Install Ballerina` section?
> > 
> > 
> > @praneesha we'll removing that section in the future right?
> 
> We will not be removing it in the release note.

+1 to add it as a note under install ballerina section of the release note.

**Comment by keizer619** [comment]
> 

+1
Also we can just mention that we support mac arm platform. No need to mention dependencies part as well

File: downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -354,6 +356,9 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.5.0](https://git
 
 ### New features
 
+#### Installers
+- Introduced a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) that supports the macOS-ARM architecture.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md:360]:
```suggestion
- Introduced a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) that supports the macOS-ARM architecture. 
```
Shouldn't we remove it from here as we are mentioning it under the `Install Ballerina` section?

**Review by praneesha — APPROVED** [review]

**Pull request merged**